src/analytics.ts

```diff
@@ -19,12 +19,33 @@ export function createAnalyticsInstance(options?: Options) {
         event_cache: Array<{ event: keyof TEvents; payload: TEvents[keyof TEvents] }> = [],
         page_view_cache: Array<{ current_page: string; platform: string; user_id: string }> = []
 
-    const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey, growthbookOptions }: Options) => {
+    const initialise = ({
+        growthbookKey,
+        growthbookDecryptionKey,
+        rudderstackKey,
+        GBAttributes = {},
+        growthbookOptions,
+    }: Options) => {
         try {
             _rudderstack = RudderStack.getRudderStackInstance(rudderstackKey)
-            if (growthbookKey && growthbookDecryptionKey) {
+            if (Object.keys(GBAttributes).length > 0)
+                core_data = {
+                    ...core_data,
+                    ...(GBAttributes.geo_location !== undefined && { country: GBAttributes.country }),
```

Comment:
[M] nit, the current logic will allow null to be added, the following should be sufficient to only allow exists value
```suggestion
                    ...(GBAttributes.geo_location && { country: GBAttributes.country }),
```

Reply:
done